import-service/test/handlers/import-products-file.test.js

```diff
@@ -0,0 +1,72 @@
+jest.mock('../../src/services/s3-client');
+
+import * as response from '../../src/services/response';
+import * as s3 from '../../src/services/s3-client';
+import { importProductsFile } from '../../src/handlers/import-products-file';
+
+import { event } from '../mocks/created-file-event.mock';
+
+beforeEach(() => {
+  response.ok = jest.fn();
```

Comment:
As far as I remember - good practice to use `jest.mock(` for module mock - it's used once at the beginning of the file than trigger it before each test case run